kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/internal/parquet/ParquetFileReaderSuite.scala

```diff
@@ -88,6 +91,132 @@ class ParquetFileReaderSuite extends AnyFunSuite
     }
   }
 
+  /**
```

Comment:
nit: just add a header saying these are the type widening tests.

Reply:
Done